src/main/java/frc/robot/subsystems/Shooter.java

```diff
@@ -1,6 +1,8 @@
 package frc.robot.subsystems;
```

Comment:
Most method names need to be rethought/combined in order to make this readable.

Reply:
done